scripts/setup_check.sh

```diff
@@ -33,12 +33,6 @@ if ! git rev-parse origin/master >& /dev/null; then
   git fetch origin
 fi
 
-# Setup mint which installs a versioned swiftformat.
-set -x
-brew install mint
-mint bootstrap --link
-
+# mint installs tools from Mintfile on demand.
 brew install clang-format
-
-clang-format -version
```

Comment:
I think having the `-version` command for `clang-format` at least was helpful for debugging purposes - I imagine we can get rid of the `swiftformat` one since we know where it comes from at least. Any reason you removed the `clang-format` one otherwise?

Reply:
`style.sh` already logs the version used. This is duplicative.